CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- URLs with double-width characters not correctly highlighting
```

Comment:
The word `correctly` is often a bit difficult in a change log since people might not know what `correct` means.

Being more concrete should help here, so this might be better:

```suggestion
- URLs ending in double-width characters not highlighting the last cell
```

Reply:
OK, that's a good point, although all double-width characters in URLs are affected, not just ones at the end